crates/autopilot/src/domain/auction/order.rs

```diff
@@ -12,9 +13,11 @@ pub struct Order {
     pub protocol_fees: Vec<fee::Policy>,
     pub side: Side,
     pub class: Class,
+    pub created_on: Option<DateTime<Utc>>,
```

Comment:
How can this be `None`?

Reply:
This struct can be constructed from the persistence layer, which is used in shadow competition where Auction with the optional order creation field is retrieved from DB using the orderbook API. 

https://github.com/cowprotocol/services/blob/71c61621e92536fb7ab69782c1debee08920e894/crates/autopilot/src/infra/shadow/orderbook/mod.rs#L19-L26

https://github.com/cowprotocol/services/blob/7fee4fab0cf46c578a800631daed1b150c6a6c0c/crates/autopilot/src/infra/persistence/dto/order.rs#L76-L108